x-pack/plugin/ccr/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/ccr/FollowIndexIT.java

```diff
@@ -44,6 +44,10 @@ public void testFollowIndex() throws Exception {
         final String leaderIndexName = "test_index1";
         if (runningAgainstLeaderCluster) {
             logger.info("Running against leader cluster");
+            Settings indexSettings = Settings.builder()
+                .put("index.soft_deletes", true)
```

Comment:
@martijnvg `index.soft_deletes` has been renamed to `index.soft_deletes.enabled` as we have one more setting under this group.

Reply:
@dnhatn Thanks, I will change this.